config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,12 @@
+# config/initializers/sidekiq.rb
+Sidekiq.default_worker_options = { 'retry' => true } # Retry indefinitely
+
```

Comment:
Line crushing when trying to run the application
```suggestion
Sidekiq.default_job_options = { 'retry' => true } # Retry indefinitely
```

Reply:
This has been resolved